### PR TITLE
Create the Crowdin Monthly's April edition

### PR DESCRIPTION
**Pull request by Junebyun**
Create 2020-05-14-do-you-speak-prestashop-april-2020.md

### CODE REVIEW

**Comment by Junebyun** [comment]
> Create 2020-05-14-do-you-speak-prestashop-april-2020.md

@LouiseBonnard request for review 😃

**Comment by Junebyun** [comment]
@ttoine request to set the date for publishing 😃

File: news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in April.
+ 
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+ 
+## Project news
+ 
+ 
+### First news, new content manager :tada:
+ 
+I would like to take this opportunity to present myself to all of you! My name is June, I joined the PrestaCrew as a Content manager last week, and I will be in charge of the Crowdin PrestaShop translation projects in the next several months. I am very excited about discovering this tool and am delighted to see that many of you from across the globe dedicate their time to these projects. I want to say thank you for your hard work, your contributions do not only help us but to anyone who is using or will be using PrestaShop ;) I look forward to communicating with you guys, if you have any questions do not hesitate to contact me anytime, I would be more than happy to answer you :smiley:
```
Comment (ttoine) [inline news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md]: In general, please try to be very neutral and incluse. The PrestaShop company is a contributor to the PrestaShop project (yes, it's confusing). That's why I suggest to remove "us" in this paragraph.

```suggestion
I would like to take this opportunity to present myself to all of you! My name is June, I joined the PrestaCrew as a Content manager last week, and I will be in charge of the Crowdin PrestaShop translation projects in the next several months. I am very excited about discovering this tool and am delighted to see that many of you from across the globe dedicate their time to these projects. I want to say thank you for your hard work, your contributions help anyone who is using or will be using PrestaShop ;) I look forward to communicating with you guys, if you have any questions do not hesitate to contact me anytime, I would be more than happy to answer you :smiley:
```

**Review by ttoine — CHANGES_REQUESTED** [review]

File: news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+ 
+### :two_hearts: Special thanks to newcomers
+ 
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, we would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated: M. Mirena (mmirena), Carlos Prince (CarlosPrince), A. JELLALI (abdelazizjellali), ymk0577, Schuster Lorenz (lorenzschuster), Goedkoopsteverf, Pedro Loureiro (pedrogrou82), Vitalie Ciobanu (VitCiobanu), Nietnagel, guarco, hismi, Micael Ferreira (micaelsf), Matas Šileikis (matassileikis), Goldylocks Portugal (goldylocks.portugal), ricardoalves, CuneytOral and Mickeycss (rockyluo). See you soon on Crowdin… and in PrestaShop :stuck_out_tongue_winking_eye:
+ 
+ 
+## A few stats
+ 
+* **52 members** were active on the project this month.
+* A total of **34,696 words have been translated** and **23,790 validated**.
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md]:
```suggestion
* A total of **57,110 words have been translated** and **23,778 validated**.
```

File: news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – April 2020 edition"
+subtitle: "Where you'll learn that PrestaShop is fully available in Danish"
+date:  2020-05-14 09:10:11
```
Comment (ttoine) [inline news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md]: Usually, Crowdin monthly are published on a Wednesday.
What do you think about next one, 20th of May 2020?

```suggestion
date:  2020-05-20 10:00:00
```

File: news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+### :two_hearts: Special thanks to newcomers
+ 
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, we would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated: M. Mirena (mmirena), Carlos Prince (CarlosPrince), A. JELLALI (abdelazizjellali), ymk0577, Schuster Lorenz (lorenzschuster), Goedkoopsteverf, Pedro Loureiro (pedrogrou82), Vitalie Ciobanu (VitCiobanu), Nietnagel, guarco, hismi, Micael Ferreira (micaelsf), Matas Šileikis (matassileikis), Goldylocks Portugal (goldylocks.portugal), ricardoalves, CuneytOral and Mickeycss (rockyluo). See you soon on Crowdin… and in PrestaShop :stuck_out_tongue_winking_eye:
+ 
+ 
+## A few stats
+ 
+* **52 members** were active on the project this month.
+* A total of **34,696 words have been translated** and **23,790 validated**.
+* All this in **35 different languages**.
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md]:
```suggestion
* All this in **34 different languages**.
```

File: news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+ 
+### :two_hearts: Special thanks to newcomers
+ 
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, we would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated: M. Mirena (mmirena), Carlos Prince (CarlosPrince), A. JELLALI (abdelazizjellali), ymk0577, Schuster Lorenz (lorenzschuster), Goedkoopsteverf, Pedro Loureiro (pedrogrou82), Vitalie Ciobanu (VitCiobanu), Nietnagel, guarco, hismi, Micael Ferreira (micaelsf), Matas Šileikis (matassileikis), Goldylocks Portugal (goldylocks.portugal), ricardoalves, CuneytOral and Mickeycss (rockyluo). See you soon on Crowdin… and in PrestaShop :stuck_out_tongue_winking_eye:
+ 
+ 
+## A few stats
+ 
+* **52 members** were active on the project this month.
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md]:
```suggestion
* **55 members** were active on the project this month.
```

File: news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+ 
+### :two_hearts: Special thanks to newcomers
+ 
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, we would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated: M. Mirena (mmirena), Carlos Prince (CarlosPrince), A. JELLALI (abdelazizjellali), ymk0577, Schuster Lorenz (lorenzschuster), Goedkoopsteverf, Pedro Loureiro (pedrogrou82), Vitalie Ciobanu (VitCiobanu), Nietnagel, guarco, hismi, Micael Ferreira (micaelsf), Matas Šileikis (matassileikis), Goldylocks Portugal (goldylocks.portugal), ricardoalves, CuneytOral and Mickeycss (rockyluo). See you soon on Crowdin… and in PrestaShop :stuck_out_tongue_winking_eye:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md]:
```suggestion
Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, we would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated: A. JELLALI (abdelazizjellali), Carlos Prince (CarlosPrince), CuneytOral, Goedkoopsteverf, Goldylocks Portugal (goldylocks.portugal), hismi, Matas Šileikis (matassileikis), Micael Ferreira (micaelsf), Mickeycss (rockyluo), Nietnagel, Pedro Loureiro, (pedrogrou82), ricardoalves, Schuster Lorenz (lorenzschuster), Vitalie Ciobanu (VitCiobanu), and ymk0577. See you soon on Crowdin… and in PrestaShop. :stuck_out_tongue_winking_eye:
```

Comment (LouiseBonnard) [reply news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md]: M. Mirena (mmirena) is not a new translator. :-)

File: news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md (hunk 1 above)
Comment (Junebyun) [reply news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md]: Ok, I'll keep that in mind next time!

File: news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md (hunk 3 above)
Comment (Junebyun) [reply news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md]: Sounds good to me!

**Review by LouiseBonnard — APPROVED** [review]

**Comment by matks** [comment]
@ttoine Can you validate this ?

**Comment by LouiseBonnard** [comment]
It is planned for tomorrow. ;-)

**Comment by LouiseBonnard** [comment]
@ttoine, the requested changes have been addressed. Can you please check and merge?

**Comment by ttoine** [comment]
done. thanks everyone.
@Junebyun welcome on board as an author of the blog 👍 😎

**Comment by Junebyun** [comment]
Yeay \o/ thanks @Antoine Thomas <antoine.thomas@prestashop.com> and @Louise
Bonnard <louise.bonnard@prestashop.com>  😍

**Pull request merged**